controllers.js

```diff
@@ -148,6 +149,24 @@ exports.deleteComment = (req, res, next) => {
     .catch(next);
 };
 
+exports.fetchAllUsers = (req, res, next) => {
+  const { sort_by } = req.query;
+  const validSortColumns = ['username', 'name'];
+
+  if (sort_by && !validSortColumns.includes(sort_by)) {
+    return res.status(400).send({ message: 'Bad Request! Invalid sort_by parameter.' });
```

Comment:
see previous feedback for info about custom errors and error handling middleware.